src/styles/card.css

```
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: flex;
+  margin: 0 auto;
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__review,
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__review {
+  line-height: 14px;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+  align-items: center;
+}
+
+.card__review_count {
+  font-size: 10px;
+  color: var(--card-main-color);
+}
+
+.price_amount {
+  line-height: 18px;
+  font-size: 16px;
+  font-weight: bold;
```

follow one style in setting font-weight
```suggestion
  font-weight: 700;
```
you can use numerical value here